### PR TITLE
docs: add service worker without PWA capabilities section

### PR DESCRIPTION
**Pull request by Dragomir-Ivanov**
Adding a separate Guide for faster reference on Custom Service worker support.

### CODE REVIEW

File: .vitepress/config.ts [hunk 1]
```diff
@@ -1,386 +1,427 @@
-import { defineConfig } from 'vitepress'
-import { withPwa } from '@vite-pwa/vitepress'
-import { version } from '../package.json'
-import { transformHead } from './scripts/transformHead'
-import { pwa } from './scripts/pwa'
+import { defineConfig } from "vitepress";
+import { withPwa } from "@vite-pwa/vitepress";
+import { version } from "../package.json";
+import { transformHead } from "./scripts/transformHead";
+import { pwa } from "./scripts/pwa";
```
Comment (userquin) [inline .vitepress/config.ts:5]: configure ESLint properly, we use single quote without semicolns

Comment (userquin) [reply .vitepress/config.ts:5]: I don't use VSCode, I cannot help you on this.

**Review by userquin — APPROVED** [review]
thx

**Comment by pastelmind** [comment]
Just wanted to say thank you for acknowledging this use case and providing a configuration setup that we can quickly copy and paste into our projects.

**Pull request merged**